### PR TITLE
Garlic Bread

### PR DESCRIPTION
**Pull request by dragoslava**
Add's garlic bread

### CODE REVIEW

**Comment by dragoslava** [comment]
!review

**Comment by alsoandanswer** [comment]
please let me change that sprite.

**Comment by Kaedwuff** [comment]
This is cheesy bread, not garlic bread =/ At least make it have butter and space spice or something that vaguely makes sense.

**Comment by Kaedwuff** [comment]
This update is getting a little feature-creepy, and the most recent thing is sort of making an inconsistency.  Garlic sauce is made from 'garlic juice' and 'corn oil', which also for some reason transitions it from a drink to a nutriment child (whose idea was that...?), and it should be garlic(juice) itself that hypothetically damages vampires, not specifically the mixture of garlic juice and corn oil.

It might be better to just leave out the vampire thing in this and focus on adding the specific food item you were adding, instead of trying to add new antag interactions.

**Comment by dragoslava** [comment]
> This update is getting a little feature-creepy, and the most recent thing is sort of making an inconsistency. Garlic sauce is made from 'garlic juice' and 'corn oil', which also for some reason transitions it from a drink to a nutriment child (whose idea was that...?), and it should be garlic(juice) itself that hypothetically damages vampires, not specifically the mixture of garlic juice and corn oil.
> 
> It might be better to just leave out the vampire thing in this and focus on adding the specific food item you were adding, instead of trying to add new antag interactions. I should also note that this will make garlic bread hurt thralls, because they are also vampires, technically, in the code.
> 
> If you really must, though, I would recommend putting the 'hurts vampires' thing under garlic juice and make garlic sauce a child of garlic juice. And make garlic juice a nutriment, I guess?

Nobody puts garlic juice on garlic bread, garlic sauce makes more sense

**Comment by ToasterStrudes** [comment]
Who the heck uses garlic juice or garlic sauce, its just butter with minced garlic ontop, baked. smh

**Comment by TehFlaminTaco** [comment]
This sprite is Cheesy Garlic Bread, and 100% does not fit the Station Aesthetic.

File: code/modules/food/recipes_oven.dm [hunk 1]
```diff
@@ -82,6 +82,16 @@
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/flatbread
 
+/datum/recipe/garlicbread
+	appliance = OVEN
+	reagents = list("spacespice" = 2, "garlicsauce" = 4)
+	items = list(
```
Comment (VTCobaltblood) [inline code/modules/food/recipes_oven.dm]: i'm sorry what? 2 cheese wedges?

**Pull request closed without merging**